vendor/bat-native-ads/src/bat/ads/internal/tokens/redeem_unblinded_payment_tokens/redeem_unblinded_payment_tokens.cc

```diff
@@ -119,6 +127,8 @@ void RedeemUnblindedPaymentTokens::OnRedeem(
 
 void RedeemUnblindedPaymentTokens::OnRedeemUnblindedPaymentTokens(
     const Result result) {
+  is_processing_ = false;
```

Comment:
Everything was working as expected, however, it made more sense to safely cover all code paths. To prove this was previously working as expected, I reverted the change and re-ran the tests.